docs/qcfractal/source/setup_server.rst

```diff
@@ -58,3 +50,49 @@ up with compute with the following lines:
 This will create a ``qcfractal-manager`` attached to the ``qcfractal-server``
 with a ProcessPoolExecutor attached.
 
+Within a Python Script
+----------------------
+
+Canonical workflows can be run from a Python script using the ``FractalSnowflake``
+instance. With default options a ``FractalSnowflake`` will spin up a database backend
+which contains no data and then destroy this database upon shudown.
+
+.. warning::
+
+    All data inside a ``FractalSnowflake`` is temporary and will be deleted when the
+    ``FractalSnowflake`` shuts down.
+
+.. code-block:: python
+
+    >>> from qcfractal import FractalSnowflake
+    >>> from qcfractal.int
```

Comment:
Not sure what this line is supposed to do?

Reply:
Accidental leftovers.